src/test/java/com/cgdp/library/validation/IsbnValidatorTest.java

```diff
@@ -0,0 +1,40 @@
+package com.cgdp.library.validation;
+
+import static com.cgdp.library.validation.IsbnValidator.isValidIsbn;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class IsbnValidatorTest {
+
+    @Test
+    void should_return_valid_isbn() {
+        String validIsbn = "9780134685991";
+        assertTrue(isValidIsbn(validIsbn));
+    }
```

Comment:
Follow exemples of other tests, separate logic by blocks
```
// given

// when

// then
```

Also use assertj-core for tests

And choose more accurate names
```suggestion
    void should_return_true_when_isbn_is_valid() {
        // given
        String validIsbn = "9780134685991";
        
        // when
        boolean result = isValidIsbn(validIsbn);
        
        // then
        assertThat(result).isTrue();
    }
```